src/main/resources/static/script.js

```diff
@@ -0,0 +1,9 @@
+// JavaScript för att visa dropdown-rutan när användaren klickar på ikonen
+document.getElementById("globe-icon").addEventListener("click", function() {
+    var dropdownContent = document.getElementById("dropdown-content");
```

Comment:
```suggestion
    let dropdownContent = document.getElementById("dropdown-content");
```
I do not think we need to reach the variable globally. Improves encapsulation with let.